server/src/defaultConfig.json

```diff
@@ -7,5 +7,11 @@
   "colorPalette": "#0078d4",
   "waitingTitle": "Thank you for choosing Lamna Healthcare",
   "waitingSubtitle": "Your clinician is joining the meeting",
-  "logoUrl": ""
+  "logoUrl": "",
+  "postCallSurveyType": "",
+  "postCallSurveyOptionsSurveyUrl": "",
+  "postCallSurveyOneQuestionPollTitle": "",
+  "postCallSurveyOneQuestionPollPrompt": "",
+  "postCallSurveyOneQuestionPollPromptType": "",
+  "postCallSurveyOneQuestionPollSaveButtonText": ""
```

Comment:
We can use the nested syntax here and provide the appropriate default values:

```
"postCall": {
    "survey": {
        "type": "onequestionpoll",
        "options": {
            "pollTitle": "Tell us how we did",
            "pollPrompt": "How satisfied are you with this virtual appointment’s audio and video quality?"
            ....
        }
    }
}
```

Reply:
As per discussion with Ralph over chat, reverted this file along with ARM changes for now since we want to include those in a separate PR once the other UI work for V2 is done and it is ready for releasing.